echopype/qc/api.py

```diff
@@ -9,32 +9,34 @@
 logger = _init_logger(__name__)
 
 
-def _clean_ping_time(ping_time_old, local_win_len=100):
-    ping_time_old_diff = np.diff(ping_time_old)
-    neg_idx = np.argwhere(
-        ping_time_old_diff < np.timedelta64(0, "ns")
-    )  # indices with negative diff
-    if neg_idx.size != 0:
-        ni = neg_idx[0][0]
-        local_win = np.arange(-local_win_len, local_win_len)
-        local_pt_diff = ping_time_old_diff[ni + local_win]
-        local_pt_median = np.median(
-            np.delete(local_pt_diff, local_win_len)
-        )  # median after removing negative element
-        ping_time_new = np.hstack(
-            (
-                ping_time_old[: ni + 1],
-                ping_time_old[ni]
-                + np.cumsum(np.hstack((local_pt_median, ping_time_old_diff[(ni + 1) :]))),
-            )
-        )
-        return _clean_ping_time(ping_time_new, local_win_len=local_win_len)
-    else:
-        return ping_time_old  # no negative diff
+def _clean_reversed(time_old: np.ndarray, win_len: float):
+    time_old_diff = np.diff(time_old)
+
+    # get indices of arr_diff with negative values
+    neg_idx = np.argwhere(time_old_diff < np.timedelta64(0, "ns")).flatten()
```

Comment:
Isn't `time_old_diff` 1D, and therefore `np.argwhere` returns a 1D array? If so, why is flatten() needed?